src/components/elements/styled-card.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Text, Header } from './typography.js'
+import device from 'themes/device'
+
+const CardWrapper = styled.article`
+    box-sizing: border-box;
+    border-radius: 6px;
+    box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.1);
+    background-color: var(--color-white);
+    padding: 4.8rem 5.6rem 2.2rem;
+
+    * {
+        max-width: 100%;
+    }
+    .card-title {
+        margin-top: 2.4rem;
+        margin-bottom: 1rem;
+    }
+    @media ${device.sm} {
+        text-align: center;
+    }
+`
+
+const StyledCard = ({ Icon, title, content_1, content_2, content_3 }) => {
```

Comment:
Can you use / combine this with `card.js`? The UI is similar

Reply:
We have basically 2 cards, this one with specific paddings/icon size and normal card. I suggest we use card.js as a regular card (how it works section).

Reply:
can that be handles with props? so we keep one card

Reply:
sure